test/resources/postman/PSR.postman_collection.json

```diff
@@ -1,9 +1,10 @@
 {
 	"info": {
-		"_postman_id": "a435773b-8698-4b2b-979b-075aebe2e292",
+		"_postman_id": "b3b1316d-bdd9-40d1-8b70-88d258135ca0",
 		"name": "PSR",
 		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
-		"_exporter_id": "1174017"
+		"_exporter_id": "17267595",
```

Comment:
Nothing relevant (payload, parameters, etc.) has changed in postman, is that expected?

Reply:
ah I think I messed up the rebase, will have a look